.github/workflows/manifest-checks.yaml

```
@@ -0,0 +1,129 @@
+name: Check manifests
+on:
+  pull_request:
+    branches:
+      - '*'
```

:thinking: is this equivalent to specifying no branches?

```suggestion
  pull_request:
```

---

[removed ](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/30/commits/ef54ed413e7e8da49ded2ba9eba875b33b1b4bc1)